app/services/qa/linked_data/language_service.rb

```diff
@@ -22,6 +24,7 @@ def literal_has_language_marker?(literal)
           def normalize_language(language)
             return language if language.blank?
             language = [language] unless language.is_a? Array
+            return nil if language.include?(WILDCARD)
```

Comment:
By setting the language to nil, there will be no filtering.  When an authority configuration setw the language to `*`, it overrides the site wide default and prevents filtering for that authority.